README.md

```diff
@@ -307,6 +307,9 @@ you will likely want to set the parameter to NoEcho in your template.
 ```yaml
 db_password:
   parameter_store: ssm_parameter_name
+saas_api_key:
+  # Parameter in a different region in the same account
+  parameter_store: us-east-1:ssm_parameter_name
```

Comment:
An alternative interface for consideration:

```yaml
saas_api_key:
  parameter_store:
    name: ssm_parameter_name
    region: us-east-1
```

Reply:
Yeah, thought about it, but wanted to keep it consistent with how we do stack output cross-region things.